src/core/Table/Table.test.tsx

```diff
@@ -2227,3 +2227,193 @@ it('should sync body horizontal scroll with header scroll', () => {
   expect(header.scrollLeft).toBe(100);
   expect(body.scrollLeft).toBe(100);
 });
+
+it.each([
+  // Dragging Name to the left of View
```

Comment:
Maybe instead of comments, add proper test case name? Beacuse now if it fails, not really clear which exactly
![image](https://user-images.githubusercontent.com/81580355/158147384-71178e54-3465-4e15-a6cf-e1d155cb3f7f.png)